### ISSUE TITLE
Change test environment API

### ISSUE THREAD

**Issue by heikomuller** [scailfin/flowserv-core#64]
The test environment to run workflows (e.g., from within Jupyter notebooks) should be simplified and be based on a full flowserv application instance. It should also provide the functionality run multiple workflows.

**Comment by heikomuller** [comment]
the test environment API should also provide the option to run the post-processing workflow separately on a set of workflow run results. For example:

```
db = Flowserv(basedir='')
wf = db.install(source='helloworld)
run_1 = wf.start_run({...})
run_2 = wf.start_run({...})
run = wf.run_postproc([run_1, run_2])
```

**Comment by heikomuller** [comment]
Closed by commit fee3d86b50c70584b470770ddfe2d27a16ec4452
